lib/taste_tester/server.rb

```diff
@@ -158,7 +160,7 @@ def start_chef_zero
         extend ::TasteTester::Windows
         start_win_chef_zero_server
       else
-        cmd = "#{chef_zero_path} --host #{@addr} --port #{@state.port} -d"
+        cmd = +"#{chef_zero_path} --host #{@addr} --port #{@state.port} -d"
```

Comment:
what is this leading plus thing?

Reply:
Marks a string as mutable (by making it an expression) when the default is frozen